data/datacatalogue3/CatalogueOntologies/Organisation roles.csv

```diff
@@ -1,5 +1,7 @@
 name
 Data originator
+Data provider
```

Comment:
can we please start adding definitions? what is difference between originator, holder and provider. I think I know but I am not sure ;-)